sonar-project.properties

```diff
@@ -4,7 +4,6 @@ sonar.host.url=https://sonar.elastic.dev
 
 sonar.sources=\
   packages, \
-  plugins, \
```

Comment:
Is this removed, because this folder only ever contains anything for external plugin devs?